openpaygo-token-generator/README.md

```diff
@@ -0,0 +1,36 @@
+This is a [Next.js](https://nextjs.org) project bootstrapped with [`create-next-app`](https://nextjs.org/docs/app/api-reference/cli/create-next-app).
```

Comment:
Could you update and clean up the `README`? Keep the build instructions, but explain here what this demo is supposed to do and remove the `nextjs` boilerplate.